### PR TITLE
ch4 language: interface grammatic

### PR DESCRIPTION
**Pull request by PatrickCQ**
as discussed at the last face to face a description of the grammatic that allows describing data models.

### CODE REVIEW

**Comment by tguild** [comment]
You and Daniel are arriving at much of the same terminology.

https://github.com/w3c/automotive/issues/306

File: spec/Gen2_Core.html [hunk 1]
```diff
@@ -137,35 +137,55 @@ <h2>Interface Grammatic</h2>
       </figure>
 	    <section data-dfn-for="tree-elements">
         <h2>Tree elements</h2>
-        <section data-dfn-for="rbranch">
-          <h3><dfn>rbranch</dfn></h3>
+        <p>
+          Tree elements are of one of two types.
+          Leafs in the tree are <a>values</a>.
```
Comment (gunnarx) [inline spec/Gen2_Core.html]: Trying to work through some of the conflicts:

This part seems to conflict with changes made in 
https://github.com/w3c/automotive/commit/76fe6eb3421a6377f450f352c43c059657a5fc22
(UlfBj  Fri Oct 18 15:07:14 2019 +0200)

somewhere around here if I'm not mistaken: https://github.com/w3c/automotive/commit/76fe6eb3421a6377f450f352c43c059657a5fc22#diff-fea9ef03786258e811be67dc6af95f24R179

Bottom line:  https://github.com/w3c/automotive/commit/76fe6eb3421a6377f450f352c43c059657a5fc22 does not seem to include the text description given here.  Do we want to include this description?

File: spec/Gen2_Core.html [hunk 2]
```diff
@@ -137,35 +137,55 @@ <h2>Interface Grammatic</h2>
...
-          <h3><dfn>actuator</dfn></h3>
-          <p>A leaf the same as <a>sensor</a> which additionaly can be changed.</p>
         </section>
-        <section data-dfn-for="element">
-          <h3><dfn>element</dfn></h3>
-          <div class="issue">to be described</div>
+        <section data-dfn-for="leaf">
+          <h3><dfn>leaf</dfn></h3>
+          <p>
+            Leafs contain the data within the interface.
```
Comment (gunnarx) [inline spec/Gen2_Core.html]: And the description that starts here (and continues for a number of lines) do we want that included?

**Comment by gunnarx** [comment]
The remaining conflicts were trivial and unimportant.
If interesting I have placed the rebased version (with all conflicts sorted) here before pushing to this

https://github.com/gunnarx/w3c-automotive/pull/1

I made a PR to myself, just to get the diffs visible.

Note: Since this is a very old PR, now placed on top of the latest, it might appear as if Patrick is trying to reverse some changes that were recently introduced, but that's not certain to be the right interpretation since these commits are from June or older!

Therefore, I'd suggest to look over which parts we want to keep or not, and then rework.

**Comment by gunnarx** [comment]
As noted above/below, this now has a merge conflict but I think it is sorted out.  But I do not know if we want the whole change as it stands - please review and discuss.

**Comment by PatrickCQ** [comment]
Working group decided on 5.11.2019 that this will be reflected in a new PR created with discussion from https://github.com/w3c/automotive/issues/306.

**Pull request closed without merging**